src/mmw/apps/modeling/views.py

```diff
@@ -235,8 +237,12 @@ def choose_worker():
     def predicate(worker_name):
         return settings.STACK_COLOR in worker_name or 'debug' in worker_name
 
+    @retry(AttributeError, delay=0.5, backoff=2, tries=3)
+    def get_list_of_workers():
+        return celery.current_app.control.inspect().ping().keys()
```

Comment:
For the sake of having descriptive error messages, consider raising an exception if `celery.current_app.control.inspect().ping()` yields no results, instead of relying on `AttributeError`.